app/models/app_remote_kill.rb

```diff
@@ -0,0 +1,77 @@
+class AppRemoteKill < ActiveRecord::Base
+  belongs_to :app
+  belongs_to :split
+
+  attribute :first_bad_version, :app_version
+  attribute :fixed_version, :app_version
+
+  validates :app, :split, :reason, :override_to, :first_bad_version, presence: true
+  validates :reason, uniqueness: { scope: %i(app split) }
```

Comment:
We can't ship and fix the same mistake twice?

Reply:
the reason exists primarily as a human identifier for mutating the existing remote kill record.

Reply:
Right, so is it that we want to make sure that you're actually changing the reason each time (instead of writing "oops" or something every time) -- i.e. a tiny reminder that you're writing this text value for other humans? I'm wondering if we would also want to enforce a minimum length too.

Reply:
oops would probably work if you were breaking a different feature each time. Yeah, I dunno, I hope that with decent example text we'll get good results. Here's what I'm drafting in the CLI docs now:

`Reason should be a camel_case slug, e.g. "home_screen_crash_jan_2019".`

Reply:
Ah, got it.